lasagne/layers/recurrent.py

```diff
@@ -222,6 +222,23 @@ def __init__(self, incoming, input_to_hidden, hidden_to_hidden,
 
         super(CustomRecurrentLayer, self).__init__(incomings, **kwargs)
 
+        input_to_hidden_layers = helper.get_all_layers(input_to_hidden)
+        input_to_hidden_in_lyrs = [lyr for lyr in input_to_hidden_layers
+                                   if isinstance(lyr, InputLayer)]
+        if len(input_to_hidden_in_lyrs) != 1:
+            raise ValueError('input_to_hidden_in_layer '
+                             'not provided and input_to_hidden does not '
+                             'have a unique InputLayer')
```

Comment:
This indicates you could provide the input layer in some way, but you can't. I'd say:
```
`input_to_hidden` must have exactly one InputLayer, but it has %d
```